cmd/nknc/commands/asset.go

```diff
@@ -0,0 +1,214 @@
+package commands
+
+import (
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/howeyc/gopass"
+	api "github.com/nknorg/nkn/v2/api/common"
+	"github.com/nknorg/nkn/v2/api/httpjson/client"
+	"github.com/nknorg/nkn/v2/common"
+	"github.com/nknorg/nkn/v2/config"
+	"github.com/nknorg/nkn/v2/util/log"
+	"github.com/nknorg/nkn/v2/vault"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	RANDBYTELEN = 4
+)
```

Comment:
It seems that this global var wasn't used at anywhere?

Reply:
yes. it was there before so I didnt remove it. this is only a change of the underlying cli lib. no logic in code has been touched. we gotta ask @yilunzhang

Reply:
No worries, you can leave it there and we will clean it up later.